index2.html

```diff
@@ -130,8 +129,6 @@ <h2>How to overcome common objections to bike lanes</h2>
 
   <div class="row">
     <div class="col-8">
-      <div class="row">
-        <div class="col-8">
```

Comment:
You need to also remove their respective closing tags, remember to use the text editor's formatter so you can see broken nesting

Reply:
hai